Veronneau-Techno-Conseil/comax-accounts
# Features/group accounts

Testing procedure:

Log in with 
wesley_van_wyk@hotmail.com
Test123!*1

Click on Network and load the contacts page. Contacts should appear and contact requests should be stated.
